CourierCore/Models/ConnectOptions.swift

```diff
@@ -29,7 +31,8 @@ public struct ConnectOptions: Equatable {
         password: String,
         isCleanSession: Bool = false,
         userProperties: [String: String]? = nil,
-        alpn: [String]? = nil
+        alpn: [String]? = nil,
+        scheme: String? = nil
```

Comment:
please assign `ssl` as the default value for scheme. 

The idea is If client of lib wants to connect using SSL, just pass "ssl", the lib won't check the port and will always connect using SSL.

Reply:
My bad, after second thoughts, i think its better to update the parameter to `useSSLWithTLS` with `Bool` type and set the default value to `false`

Reply:
Done mas. I just added the `useSSLWithTLS` variable in ConnectOptions